.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: php
+php:
+  - 7.0
+  - 5.6
+  - 5.5
+matrix:
+  allow_failures:
+    - php: 7.0 # This should be fixed, temporary for the PHP7 support badge
+install:
+  - composer self-update
+  - composer install
+script:
+  - vendor/bin/robo test
```

Comment:
we should probably add https://docs.codeclimate.com/v1.0/docs/travis-ci-php-test-coverage once we make the switch to codeclimate.